### PR TITLE
header handles flags, especially the fuzzy flag

### PR DESCRIPTION
**Pull request by dfherr**
i'm not 100% sure, please double check, but this should fix the fuzzy header issue

### CODE REVIEW

File: spec/poparser/po_spec.rb [hunk 1]
```diff
@@ -102,6 +102,23 @@
       expect(@po.header.comments).to eq(["Arash Mousavi <mousavi.arash@gmail.com>, 2014.", ""])
     end
 
+    it 'should parse the flag and be fuzzy' do
+      puts @po.header.flag.inspect
+      puts @po.header.flag.class
```
Comment (arashm) [inline spec/poparser/po_spec.rb]: Please remove these two `puts` lines.

File: lib/poparser/header.rb [hunk 2]
```diff
@@ -9,6 +9,7 @@ def initialize(entry)
       @entry            = entry
       @comments         = entry.translator_comment.to_s
       @original_configs = convert_msgstr_to_hash(entry.msgstr)
+      @flag = entry.flag
```
Comment (arashm) [inline lib/poparser/header.rb]: Please align the equal with others.

File: lib/poparser/header.rb [hunk 3]
```diff
@@ -23,15 +24,41 @@ def configs
       @original_configs.merge(hash)
     end
 
+    # Checks if the entry is fuzzy
+    #
+    # @return [Boolean]
+    def fuzzy?
+      @flag.to_s.match('fuzzy') ? true : false
+    end
+
```
Comment (arashm) [inline lib/poparser/header.rb:33]: Extra new line here. Please remove one of them.

**Review by arashm — CHANGES_REQUESTED** [review]

**Comment by dfherr** [comment]
added the requested changes

**Pull request merged**